dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/repositories/ReviewQueueRepository.kt

```diff
@@ -16,4 +16,9 @@ interface ReviewQueueRepository : JpaRepository<ReviewQueueEntity, String> {
             "ORDER BY status.receptionTime ASC",
     )
     fun getSortedPendingDataIds(): List<String>
+
+    /**
+     * Deletes all queued QA requests for a specific dataId.
```

Comment:
Ist nicht nur ein einziger QA request in der queue?
Also wird nicht nur dieser eine einzige gelöscht?

Die Beschreibung klingt so, als könnten da auch mehrere sein.

Reply:
Angepasst